docker-app/qfieldcloud/core/utils2/storage.py

```diff
@@ -76,7 +76,7 @@ def file_response(
         response["redirect_uri"] = url
 
         return response
-    elif settings.DEBUG:
+    elif settings.DEBUG or settings.IN_TEST_SUITE:
```

Comment:
See discussion on #267

IMO this moves us away from testing the actual production stack (e.g. we wouldn't catch anymore bugs in the code just above and/or nginx config). Ideally we should find another way (either have download implementation that works without Nginx, or run the tests against the running docker compose stack)

Reply:
Yes, we have this difference between prod and test environment. However, downloading files is such a basic functionality, that 1) we would see it does not work immediately 2) we have semi integration tests, that are mostly focused on Django so far, so we don't test nginx at the moment anyway.

> either have download implementation that works without Nginx,

I am -1 for this, since I am the maintener of the client side of QFieldCloud, I would reduce networking on the clients at all costs. In this case it is proxying trough nginx which is already pretty good at proxying.

> run the tests against the running docker compose stack

I am +0.5 for this, because we would lose the debugger functionality, as all requests will be proxied with `nginx`.

The best of both worlds would be to have a dedicated integration tests for downloading files that uses `nginx` and we keep all the rest the same.